spec/schema.json

```diff
@@ -0,0 +1,209 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Companies House - Limited Partnerships API",
+    "description": "An API to allow the registration, transition and update of limited partnerships",
+    "version": "1.0.0"
+  },
+  "host": "api.companieshouse.gov.uk",
+  "tags": [
+    {
+      "name": "Limited Partnerships"
+    }
+  ],
+  "paths": {
+    "/transactions/{transaction_id}/limited-partnership/partnership": {
```

Comment:
I think only the healthcheck end-point should feature in the spec for the moment. We add the others when we come to implement the end-points in controllers.

Reply:
Cut this back.